osu.Game/Overlays/Dashboard/Dashboard/DashboardBeatmapPanel.cs

```diff
@@ -0,0 +1,164 @@
+            {
+                background = new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = ColourProvider.Background3,
+                    Alpha = 0
+                },
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Padding = new MarginPadding { Horizontal = 10 },
+                    Child = new GridContainer
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        ColumnDimensions = new[]
+                        {
+                            new Dimension(GridSizeMode.Absolute, 70),
+                            new Dimension(),
+                            new Dimension(GridSizeMode.AutoSize)
+                        },
+                        RowDimensions = new[]
+                        {
+                            new Dimension()
+                        },
+                        Content = new[]
+                        {
+                            new Drawable[]
+                            {
+                                new Container
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Masking = true,
+                                    CornerRadius = 6,
+                                    Child = new UpdateableBeatmapSetCover
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Anchor = Anchor.Centre,
+                                        Origin = Anchor.Centre,
+                                        BeatmapSet = SetInfo
+                                    }
+                                },
+                                new Container
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Padding = new MarginPadding { Horizontal = 10 },
+                                    Child = new FillFlowContainer
+                                    {
+                                        Anchor = Anchor.CentreLeft,
+                                        Origin = Anchor.CentreLeft,
+                                        AutoSizeAxes = Axes.Y,
+                                        RelativeSizeAxes = Axes.X,
+                                        Direction = FillDirection.Vertical,
+                                        Children = new Drawable[]
+                                        {
+                                            new OsuSpriteText
+                                            {
+                                                RelativeSizeAxes = Axes.X,
+                                                Truncate = true,
+                                                Font = OsuFont.GetFont(weight: FontWeight.SemiBold),
```

Comment:
web-side these are all `Regular` but using that here is pretty bad due to font AA

would have been nice if you had mentioned that (if you were aware)

Reply:
AFAIK font weights for some reason are offset by one (600 here looks like 500 on web), or at least that was the case last time I checked - quite a few overlay elements have 'wrong' font weights in place to match web visually. Font AA probably makes it a bit harder to compare, too.

might be the reason for the different weight being chosen here.

Reply:
doubt it, as other weights used in other places in this PR match (i.e. the `Bold` usages)

Reply:
We should probably keep these consistent, even if it looks bad for the time being, and figure out a solution that works universally in the future.